CMakeLists.txt

```diff
@@ -15,7 +15,8 @@ set( CMAKE_INCLUDE_CURRENT_DIR ON )
 include_directories( program/src/ )
 
 # gcc compiler/linker flags
-add_compile_options( -ggdb -Wall -Wextra -Werror -m64 )
+# FIXME: add -Werror back
+add_compile_options( -ggdb -Wall -Wextra -m64 )
```

Comment:
This should no longer be necessary once we merge https://github.com/pyth-network/pyth-client/pull/89.